deployments/ocis-monitoring/README.md

```diff
@@ -0,0 +1,48 @@
+# oCIS with monitoring deployment example
+
+## Introduction
+
+This example shows how to deploy oCIS with monitoring, in this case Grafana.
+
+***Note***: This example is not intended for production use. It is intended to get a working oCIS
+with Grafana running in Kubernetes as quickly as possible. It is not hardened in any way. This also applies to password and http usage, so the password for Grafana at the beginning of the `helmfile` should be changed. This example uses unencypted HTTP between the Grafana-Agent and mimir/loki/tempo.
+
+## Getting started
+
+### Prerequisites
+
+This example requires the following things to be installed:
+
+- [Kubernetes](https://kubernetes.io/) cluster, with an ingress controller installed.
+- [Helm](https://helm.sh/) v3
+- [Helmfile](https://github.com/helmfile/helmfile)
+
+### End result
+
+After following the steps in this guide, you should be able to access the following endpoint, you
+may want to add these to your `/etc/hosts` file pointing to your ingress controller IP:
+
+- https://ocis.kube.owncloud.test
+- https://grafana.kube.owncloud.test
+
+Note that if you want to use your own hostname and domain, you will have to change the `externalDomain` value in oCIS and the Domains for Loki, Mimir and Tempo at the beginning of the `helmfile`. Please also ensure that the hostnames for Loki, Mimir and Tempo can be resolved within the cluster.
+
+### Deploying
+
+In this directory, run the following commands:
+
+```bash
+$ helmfile sync
+```
+
+This will deploy oCIS and Grafana.
+
+### Logging in
+
+You can get the admin password with the following command:
```

Comment:
we probably should add that for grafana, too:

`kubectl -n grafana get secrets/grafana --template='{{ index .data "admin-password" | base64decode | printf "%s\n" }}'`